bootstrap-workshop.sh

```diff
@@ -14,33 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Variables
+set -e
 
 if [[ $OSTYPE == "linux-gnu" && $CLOUD_SHELL == true ]]; then 
 
-    export PROJECT=$(gcloud config get-value project)
-    export BASE_DIR=${BASE_DIR:="${PWD}"}
-    export WORK_DIR=${WORK_DIR:="${BASE_DIR}/workdir"}
-
-    echo "WORK_DIR set to $WORK_DIR"
-    gcloud config set project $PROJECT
-
-    source ./common/settings.env
     ./common/install-tools.sh
-    echo -e "\nMultiple tasks are running asynchronously to setup your environment.  It may appear frozen, but you can check the logs in $WORK_DIR for additional details in another terminal window." 
 
-    ./gke/provision-gke.sh &> ${WORK_DIR}/provision-gke.log &
-    ./connect-hub/provision-remote-gce.sh &> ${WORK_DIR}/provision-remote.log &
-    wait
+    ./gke/provision-gke.sh
+    ./connect-hub/provision-remote-gce.sh
```

Comment:
Not sure, will revert back and check with Amina on Monday.

Reply:
@cgrant The fact that they were async was causing the script to continue even if one of them fails. In order to manage the async commands and have them fail if one of them does it required a lot more work that was in my opinion unnecessary. 
When I ran the script it took less than 8 mins all in all. I understand that different connections/loads will cause it to be slower but the way the Study Jam is set up is that the bootstrap script is run then the Anthos overview is given which makes it not necessary for the script to be super fast.

I thought it would be better for the script to fail when appropriate than to mistakenly think both clusters were created successfully and then run into multiple errors with later steps.

These are my thoughts, what do you think?